### PR TITLE
To handle issue after big delete, since region merge feature is already released, update on it

### PR DESCRIPTION
**Pull request by Tetsuya3850**
### What is changed, added or deleted? (Required)
It seems the docs has been a bit stale since we released the region merge feature.
Thus, update on it and present an appropriate link for more details

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Tetsuya3850** [comment]
@purelind 
Hi, thank you for checking this. The ok to test was added but I don't seem to see any changes to the test status.

**Comment by Tetsuya3850** [comment]
@purelind
Hi, much appreciated if you can please take a look at this at your convenience 🙇

**Comment by Tetsuya3850** [comment]
@ran-huang 
Hi, much appreciated if you can please take a look at this at your convenience 🙇

**Comment by ran-huang** [comment]
Thank you for updating this! @Frank945946 PTAL

File: faq/migration-tidb-faq.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ Yes.
 
 ### Why does the query speed getting slow after deleting data?
 
-Deleting a large amount of data leaves a lot of useless keys, affecting the query efficiency. Currently the Region Merge feature is in development, which is expected to solve this problem. For details, see the [deleting data section in TiDB Best Practices](https://en.pingcap.com/blog/tidb-best-practice/#write).
+Deleting a large amount of data leaves a lot of useless keys, affecting the query efficiency. [Region Merge feature](/best-practices/massive-regions-best-practices.md#method-3-enable-region-merge) should solve this problem. For more details, see the [deleting data section in TiDB Best Practices](https://en.pingcap.com/blog/tidb-best-practice/#write).
```
Comment (ran-huang) [inline faq/migration-tidb-faq.md:173]:
```suggestion
Deleting a large amount of data leaves a lot of useless keys, affecting the query efficiency. The [Region Merge feature](/best-practices/massive-regions-best-practices.md#method-3-enable-region-merge) should solve this problem. For more details, see the [deleting data section in TiDB Best Practices](https://en.pingcap.com/blog/tidb-best-practice/#write).
```

**Review by ran-huang — APPROVED** [review]
rest LGTM

File: faq/migration-tidb-faq.md (hunk 1 above)
Comment (Frank945946) [reply faq/migration-tidb-faq.md:173]:
```suggestion
Deleting a large amount of data leaves a lot of useless keys, affecting the query efficiency. [Region Merge feature](/best-practices/massive-regions-best-practices.md#method-3-enable-region-merge) can solve this problem. For more details, see the [deleting data section in TiDB Best Practices](https://en.pingcap.com/blog/tidb-best-practice/#write).
```

**Review by Frank945946 — APPROVED** [review]

**Pull request merged**